installer/dbscript/dbscript-2.sql

```diff
@@ -1880,8 +1880,10 @@ INSERT INTO `ohrm_user_role_data_group` (`user_role_id`, `data_group_id`, `can_r
 (4, 49, 1, 0, 0, 0, 0),
 
 (1, 50, 1, 0, 0, 0, 0),
+(1, 50, 1, 0, 0, 0, 1),
 (2, 50, 0, 0, 0, 0, 0),
 (3, 50, 1, 0, 0, 0, 0),
+(3, 50, 0, 0, 0, 0, 1),
```

Comment:
rebase branch, and need to  move these to migration file

after rebase, open `installer/Migration/V5_0_0/Migration.php`
add below code end of the `up` method,

```php
        $this->getDataGroupHelper()->addDataGroupPermissions(
            'time_employee_reports',
            'Admin',
            true,
            false,
            false,
            false,
            true
        );
        $this->getDataGroupHelper()->addDataGroupPermissions(
            'time_employee_reports',
            'Supervisor',
            false,
            false,
            false,
            false,
            true
        );
```

Reply:
done